pkg/client/group_mgmt_client.go

```diff
@@ -31,6 +32,7 @@ type GroupMgmtClient struct {
 	WaitOnJob    bool
 	Username     string
 	Password     string
+	TenantAware  bool
```

Comment:
Would "IsTenant" be more appropriate?  This patch seems to make the SDK tenant-aware, but this settings seems to indicate whether we are creating a client instance on behalf of a tenant user.  Same thing at line 65, etc.